sdk/client/src/stream/streaming_client.rs

```diff
@@ -206,12 +183,30 @@ impl StreamingClient {
         Ok(subscription_stream)
     }
 
+    pub async fn unsubscribe(&mut self, id: Id){
+        match self.subscriptions.read().await.get(&id){
+            None => {}
+            Some(_) => {}
+        }
+    }
```

Comment:
This function seems to do nothing? You'd want to remove it from the hashmap instead of just querying for it right?